### DISCUSSION TITLE
Event updates "accept all" fails with status 80004005

### DISCUSSION THREAD

**Discussion by henning** [kewisch/gdata-provider#444] [Support]
Not sure if i shall open a bug or discuss it first here, please let me know if its wrong here...

I have got a mail for a change to a repeating event.

The diff is displayed nicely as expected, and I am offered a button to "accept all" 

On pressing that button, i get a popup window saying:

Processing message failed. Status: 80004005.

Is there anything i can do?

Version information:

Thunderbird 91.9.1 on Ubuntu 22.04
Google Calendar provider 102.0.0

Let me know if there is any further info required to help getting this solved.

**Comment by walterzanette** [comment]
Hi Henning, 

I added some info to the issue that could be relevant for your diagnose. If you need a machine where the issue can be reproduced just contact me an I'll let you remote in.
